lib/db/mysql/index.js

```diff
@@ -431,7 +430,67 @@ MysqlStore.prototype = {
   getActiveClientTokensByUid: function getActiveClientTokensByUid(uid) {
     return this._read(QUERY_ACTIVE_CLIENT_TOKENS_BY_UID, [
       buf(uid)
-    ]);
+    ]).then(function(activeClientIds) {
+      var activeClients = {};
```

Comment:
do you think we can make a single function that will work for both (memory and MySQL engines)?